SCRWebService/endpoints/tests/test_summary_care_record.py

```diff
@@ -0,0 +1,59 @@
+"""Tests associated with the Summary Care Record endpoint"""
+import json
+import pathlib
+
+from builder.pystache_message_builder import MessageGenerationError
+from utilities import file_utilities
+from definitions import ROOT_DIR
+from tornado.testing import AsyncHTTPTestCase
+from tornado.web import Application
+from unittest import mock
+from endpoints import summary_care_record
+
+GP_SUMMARY_UPLOAD_URL = "/gp_summary_upload"
+
+complete_data_path = pathlib.Path(ROOT_DIR) / 'endpoints' / 'tests' / 'data' / 'complete_input.json'
+
+
+class TestSummaryCareRecord(AsyncHTTPTestCase):
+    """Tests associated with the Summary Care Record endpoint"""
+    
+    def get_app(self):
+        self.forwarder = mock.MagicMock()
+
+        return Application([
+            (r'/gp_summary_upload', summary_care_record.SummaryCareRecord,
+             dict(forwarder=self.forwarder))
+        ])
+
+    def test_handler_returns_message_processing_response(self):
+        body = file_utilities.FileUtilities.get_file_dict(complete_data_path)
+        self.forwarder.forward_message_to_mhs.return_value = "Nice response message"
```

Comment:
if we are going to mock things, we should assert the mock is invoked with the expected parameters (so we know the rest layer is deserialising the jSON correctly). This is also why i dont like mocking.